app/controllers/analytics/districts_controller.rb

```diff
@@ -82,6 +83,12 @@ def set_dashboard_analytics(period, prev_periods)
       end
   end
 
+  def set_facility_keys
+    facilities = @organization_district.facilities.order(:name).pluck(:id, :name).to_h
+
+    @facility_keys = { total: "Total" }.merge(facilities).with_indifferent_access
+  end
```

Comment:
We pre-fetch a list of facilities in alphabetical order to help generate the rows of data in the CSV in order.